app/scripts/modules/core/src/pipeline/config/triggers/TriggersPageContent.tsx

```diff
@@ -0,0 +1,144 @@
+}
+
+export function TriggersPageContent(props: ITriggersPageContentProps) {
+  const showProperties = SETTINGS.feature.quietPeriod || SETTINGS.feature.managedServiceAccounts;
+  const { pipeline, application, updatePipelineConfig } = props;
+
+  function updateRoles(roles: any[]): void {
+    updatePipelineConfig({ roles });
+  }
+
+  function addTrigger(): void {
+    const triggerTypes = Registry.pipeline.getTriggerTypes();
+    let newTrigger: ITrigger = { enabled: true, type: null };
+    if (triggerTypes.length === 1) {
+      newTrigger = { enabled: true, type: triggerTypes[0].key };
+    }
+    updatePipelineConfig({ triggers: [...pipeline.triggers, newTrigger] });
+  }
+
+  function removeTrigger(triggerIndex: number): void {
+    const triggers = pipeline.triggers.slice(0);
+    triggers.splice(triggerIndex, 1);
+    updatePipelineConfig({ triggers });
+  }
+
+  function updateTrigger(index: number, changes: Partial<ITrigger>) {
+    const triggers = pipeline.triggers.slice(0);
+    extend(triggers[index], changes);
+    PipelineConfigValidator.validatePipeline(pipeline);
+    updatePipelineConfig({ triggers });
+    if (SETTINGS.feature['artifactsRewrite']) {
+      removeUnusedExpectedArtifacts(pipeline);
+    }
+  }
+
+  // Expected Artifacts
+  function updateExpectedArtifacts(expectedArtifacts: IExpectedArtifact[]) {
+    updatePipelineConfig({ expectedArtifacts });
+  }
+
+  function removeUnusedExpectedArtifacts(pipelineParam: IPipeline) {
+    // remove unused expected artifacts from the pipeline
+    const expectedArtifacts: IExpectedArtifact[] = [...pipelineParam.expectedArtifacts];
+    const newExpectedArtifacts: IExpectedArtifact[] = expectedArtifacts.slice(0);
+    expectedArtifacts.forEach(expectedArtifact => {
+      if (
+        !pipelineParam.triggers.find(t => t.expectedArtifactIds && t.expectedArtifactIds.includes(expectedArtifact.id))
+      ) {
+        newExpectedArtifacts.splice(findIndex(newExpectedArtifacts, e => e.id === expectedArtifact.id), 1);
+      }
+      ArtifactReferenceService.removeReferenceFromStages(expectedArtifact.id, pipelineParam.stages);
+    });
+    updatePipelineConfig({ expectedArtifacts: newExpectedArtifacts });
+  }
+
+  return (
+    <div className={'triggers'}>
+      {pipeline.triggers.length > 0 && showProperties && (
+        <div className="form-horizontal panel-pipeline-phase">
+          <div className="form-group row">
+            <div className="col-md-12">
+              <div className="trigger-config">
+                {SETTINGS.feature.managedServiceAccounts && (
+                  <PipelineRoles roles={pipeline.roles} updateRoles={updateRoles} />
+                )}
+                {SETTINGS.feature.quietPeriod && (
+                  <div className="row">
+                    <div className="col-md-10">
+                      <div className="col-md-9 col-md-offset-3">
+                        <FormField
+                          name={'respectQuietPeriod'}
```

Comment:
```suggestion
                          name="respectQuietPeriod"
```

Reply:
there are a few other places in this file where this change needs to be made too